packages/application-components/src/theming.ts

```diff
@@ -8,33 +8,6 @@ import {
 // https://github.com/commercetools/merchant-center-application-kit/pull/3353
 export const themesOverrides = {
   default: {
```

Comment:
After reviewing [the PR](https://github.com/commercetools/merchant-center-application-kit/pull/3414/files#diff-e98b3ff7bf77917ad89bcfae2e36182b3fcbcd90fbb148ae21fcfaa93b4ac30f) where we introduced the recolouring in `app-kit` and looking for usages of the remaining tokens here, I found there's a bunch of them that are use only in one place. For those ones, as we did in `ui-kit`, we should remove them and moved their values to the only place where they are needed.

Reply:
Thanks @CarlosCortizasCT for the input! I made some changes here - [3b8650a](https://github.com/commercetools/merchant-center-application-kit/pull/3497/commits/3b8650a3408847a15d3fa790816717fb35d147a0). Let me know if that's good now.